src/models/user.ts

```diff
@@ -30,13 +30,13 @@ export class User {
   userType: UserType;
 
   @Column({ nullable: true })
-  institutionId!: number;
+  institutionId?: number | null;
```

Comment:
Isn't that the whole point of using the nullable in the decorator?
I'm not really sure though, you might need this for type safety

Reply:
yeah that's why I've added it